README.md

```diff
@@ -173,7 +184,9 @@ helm upgrade --install -n stackrox --create-namespace stackrox-secured-cluster-s
   --set centralEndpoint="$CENTRAL_ENDPOINT"
 ```
 
-If you're installing on a single node cluster, or the default installation results in pods stuck pending due to lack of resources, use the following command instead to reduce stackrox-secured-cluster-services resource requirements. Keep in mind that these reduced resource settings are not suited for a production setup.
+#### Secured Cluster Services Installation in Clusters With Limited Resources
+
+If you're installing in a single node cluster, or the default installation results in pods stuck pending due to lack of resources, use the following command instead to reduce stackrox-secured-cluster-services resource requirements. Keep in mind that these reduced resource settings are not suited for a production setup.
```

Comment:
The change here is the 4th word: "on" vs "in".  I realized the headers I had removed used "installation _in_", so I changed this new text to match that.